danger_code_ws/2.cpp

```diff
@@ -0,0 +1,87 @@
+#include <iostream>
+#include <cstdlib>
+
+// Function to dynamically allocate a matrix of size rows x cols
+double** allocateMatrix(int rows, int cols) {
```

Comment:
Function names should use camel case (e.g., `allocateMatrix` should be `allocateMatrix`).  Consistent naming improves readability.